tests/integration/bond_test.py

```diff
@@ -187,7 +200,11 @@ def test_add_and_remove_bond_with_port_config(eth1_up, eth2_up):
 def test_add_bond_with_port_config_and_modify(eth1_up, eth2_up):
     state = yaml.load(BOND99_PORT_YAML_BASE, Loader=yaml.SafeLoader)
     try:
-        libnmstate.apply(state)
+        apply_with_description(
+            "Add bond interface bond99 with bonding mode active-backup, "
+            "attach two ports eth1 and eth2 with ports config",
```

Comment:
What is "ports config" supposed to mean?

Reply:
I replied in the above conversation.